src/main/scala/com/github/conduktor/ksm/source/FileSourceAcl.scala

```diff
@@ -13,6 +14,7 @@ class FileSourceAcl(parserRegistry: AclParserRegistry)
 
   var lastModified: Long = -1
```

Comment:
no longer needed ?

Reply:
yeah, somehow Intellij is not showing me that it's unused so I missed it